src/Sniff/PunctuationSpacingSniff.php

```diff
@@ -35,20 +72,16 @@ protected function shouldHaveSpaceAfter(int $tokenPosition, array $tokens): ?int
     {
         $token = $tokens[$tokenPosition];
 
-        if ($this->isTokenMatching($token, Token::PUNCTUATION_TYPE, self::NO_SPACE_AFTER)) {
-            return 0;
+        if (!$this->isTokenMatching($token, Token::PUNCTUATION_TYPE)) {
+            return null;
         }
 
-        if ($this->isTokenMatching($token, Token::PUNCTUATION_TYPE, self::ONE_SPACE_AFTER)) {
-            // We cannot add one space after, if the next token need zero space before: `[1,2,3,]`.
-            $nextPosition = $this->findNext(Token::WHITESPACE_TOKENS, $tokens, $tokenPosition + 1, true);
-            if (false !== $nextPosition && null !== $this->shouldHaveSpaceBefore($nextPosition, $tokens)) {
-                return null;
-            }
-
-            return 1;
+        // We cannot change spaces after one token, if the next one have a constraint: `[1,2,3,]`.
```

Comment:
`// We cannot change spaces after a token if the next one has a constraint: `[1,2,3,]`.`

Nitpicking, I know :)